hostutil/host_remote.go

```diff
@@ -37,7 +37,7 @@ func RunRemoteScript(ctx context.Context, h *host.Host, script string, sshOption
 		cmdArgs = append(cmdArgs, "sudo")
 	}
 
-	cmdArgs = append(cmdArgs, "sh", script)
+	cmdArgs = append(cmdArgs, "sh", "-c", fmt.Sprintf("'%s'", script))
```

Comment:
I foresee a terrible shell quoting bug

Reply:
We don't use it for actual shell scripts.

Reply:
yet.

Reply:
Added a comment.